scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/LegacyScanner.scala

```diff
@@ -695,10 +695,13 @@ class LegacyScanner(input: Input, dialect: Dialect) {
         setStrVal()
         token = STRINGLIT
       }
-    } else if (ch == '\\') {
-      nextRawChar()
+    } else if (ch == '\\' && !multiLine) {
```

Comment:
what happens if `multiline = true`?

Reply:
The difference is that for `multiline = true` having `\` at the end of is allowed.
This affect only closing quotes handling: `\"""` and `\"`
```scala
raw"""ab\""" // ok
raw"ab\"     // error - unclosed string literal
```